docs/python-env.md

```diff
@@ -0,0 +1,119 @@
+# Python Environment Considerations
+
+## Introduction
+
+Many of the currently available plugins for `python-lsp-server` need to run in or access your project's Python environment (virtualenv/venv) in order to work correctly, while others at least benefit from it (cf. the listing below). There are several ways to achieve this, which we'll discuss here.
+
+
+## List of plugins
+
+Here is a non-exhaustive list of plugins with information relevant to this discussion (credit goes to [David Vicente](https://github.com/david-vicente) for compiling the original list this is based on[^discussion]):
```

Comment:
@david-vicente In case this is accepted, I hope it's fine to credit you here?

Reply:
Sure, go ahead. Thanks!